eland/operations.py

```diff
@@ -285,6 +287,45 @@ def _metric_aggs(
                             results[field] = response["aggregations"][
                                 "percentiles_" + field
                             ]["values"]["50.0"]
+                        elif func[1] == "variance":
```

Comment:
We'll also need to add these transformations into `DataFrame.agg()` until #179 is resolved. Definitely feeling the pain until that refactor.

Reply:
++ on refactor - getting pretty messy